tests/python/unittest/test_qnn_ops.py

```diff
@@ -0,0 +1,260 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import tvm
+import numpy as np
+from tvm import relay
+from tvm.relay.testing import create_workload
+from tvm.contrib import graph_runtime
+
```

Comment:
Are you planning in more of your qnn work to keep adding more unit tests to this one file or would you be adding separate files  ? If not, then please consider renaming this to test_qnn_requantize.py ? 

One of the things we will find as we keep adding tests is that it takes a long time to test one change as all the remaining tests need to be run. I don't yet grok the test framework but having multiple smaller tests might help us with test level parallelism considering the larger core counts these days. However that's an observation and not a requirement.

Reply:
Having separate files seems like the right thing to do. Will rename the file.